api/src/paths/project/{projectId}/survey/create.ts

```diff
@@ -202,7 +202,8 @@ POST.apiDoc = {
                   type: 'string'
                 },
                 field_method_id: {
```

Comment:
It's probably in our best interest to remove and field mthod code entirely, and not just allow it to be null. Thoughts?

Reply:
Yeah I agree, I think it could cause some confusing leaving this in here.

Reply:
good call, removed field method from the open api specs

Reply:
> good call, removed field method from the open api specs

What I mean is, we should probably remove any and all mention of `field_method` from the codebase. Example:

![image](https://github.com/bcgov/biohubbc/assets/18599267/e02d96c8-aa10-4ebf-926c-3a182a6381df)

Reply:
I have since removed field method from the project 😅